src/garvis/cli.py

```
@@ -105,11 +116,67 @@ def _run_local_lattice_cycle(args: argparse.Namespace) -> int:
+            continue
+
+        print("GARVIS:")
+        print(reply)
+
+
+def _run_local(args: argparse.Namespace) -> int:
+    _configure_local_memory(args)
+
+    try:
+        runtime = _build_local_runtime()
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)